pod/perldelta.pod

```diff
@@ -414,8 +414,12 @@ the perldelta of a previous release.
 
 =head1 Obituary
 
-XXX If any significant core contributor or member of the CPAN community has
-died, add a short obituary here.
+David H. Adler (DHA) passed away on November 16, 2021.  David co-founded the
```

Comment:
I think I'd say something like "co-founded NY.pm, the first Perl users group, and later co-founded Perl mongers.

He may have a had in organizing the New York City Perl Meetup, but I wasn't a part of that effort so my memories are fuzzy on that. Charlie Gonzales might know.

He also administered the White Camel Awards for several years.